ci-automation/ci-config.env

```diff
@@ -129,3 +129,7 @@ AWS_PARALLEL="${PARALLEL_TESTS:-8}"
 : ${AZURE_VNET_SUBNET_NAME:="jenkins-vnet-westeurope"}
 AZURE_PARALLEL="${PARALLEL_TESTS:-20}"
 AZURE_LOCATION="${AZURE_LOCATION:-westeurope}"
+
+# -- Openstack --
+: ${OPENSTACK_IMAGE_NAME:='flatcar_production_openstack_image.img.gz'}
```

Comment:
Did we port the `.gz` image generation flatcar-3033? I guess it may be missing for LTS-2021 but that's ok.

Reply:
It seems to be available on flatcar-3033: https://github.com/flatcar/scripts/commit/787e7689b88e5cdae3ba4facf30601ab5baac9e3